index.html

```diff
@@ -12,7 +12,7 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 </head>
 <a href="https://github.com/ILUGD/ILUGD.github.io" target = "_blank"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/38ef81f8aca64bb9a64448d0d70f1308ef5341ab/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6461726b626c75655f3132313632312e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png"></a>
-<body class="page-home">
+<body class="page-home" background="assets/img/delhi.jpg">
```

Comment:
deprecated. use css instead

Reply:
![image](https://user-images.githubusercontent.com/25998496/48675397-68b68980-eb7e-11e8-9d5a-becb7ec96b0a.png)

this is the preview of  the background

Reply:
Please use CSS